frontend/src/components/style/variables.scss

```diff
@@ -0,0 +1,5 @@
+// COLORS
+$mainPurple: #8d20ae;
```

Comment:
I'd advice against using color names in variable names : if your client wants to transition the main color to YellowGreen ( <3 ), you'll have something like `$mainPurple: yellowgreen`... ಠ_ಠ